source/renderer/app/analytics/sendPageNavigationEventOnRender.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component, ComponentClass } from 'react';
+import { inject } from 'mobx-react';
+import { InjectedProps } from '../types/injectedPropsType';
+
+export function sendPageNavigationEventOnRender(
```

Comment:
Try to do that with React-Router components (centrally in Routes.tsx) instead having to use decorator for every page component.